modern/src/ReloadPrompt.tsx

```diff
@@ -0,0 +1,56 @@
+import { Snackbar, Button } from '@mui/material';
+import React from 'react'
+import { useTranslation } from './common/components/LocalizationProvider';
+import { useAttributePreference } from './common/util/preferences';
+import { useRegisterSW } from 'virtual:pwa-register/react'
+
+// Based on https://vite-pwa-org.netlify.app/frameworks/react.html
+function ReloadPrompt() {
+  const t = useTranslation();
+
+  const serviceWorkerUpdateInterval = useAttributePreference('serviceWorkerUpdateInterval', 3600000);
```

Comment:
This should be server only. Also we usually have values in seconds or milliseconds. This seems to be in hours?

Reply:
I'm still not 100% sure I understand different settings and possibilities in Traccar. Are we talking about here checking only ```state.session.server.attributes.serviceWorkerUpdateInterval``` right? or are we talking about traccar.conf on the server side?
Also value is in milliseconds 60 * 60 * 1000 = 3600000

Reply:
We're talking about `state.session.server.attributes.serviceWorkerUpdateInterval`.